config.json

```diff
@@ -56,6 +56,18 @@
         "concepts": [
           "numbers"
         ]
+      },
+      {
+        "slug": "annalyns-infiltration",
+        "name": "Annalyn's Infiltration",
+        "uuid": "9050f8c3-9976-4c4d-93e7-c9c549b1c524",
+        "prerequisites": [
+          "basics"
+        ],
+        "status": "beta",
```

Comment:
The status of the exercises should probably be `wip` until the concept and exercise have both been completed and merged.

Reply:
This won't apply when I make Annalyn's Infiltration a practice exercise. Right?

Reply:
I don't think Annalyn's infiltration as suitable as a practice exercise. @ErikSchierboom do you have an opinion on this?

Reply:
No, not really. It has been designed as a concept exercise and is therefore quite basic. I don't think it would make for a good practice exercise. It would also be weird for students to have it be a practice exercise here and a concept exercise in other tracks.

Reply:
Ok. I'll close this PR then and open the next concept as an issue for discussion and coordination.

Reply:
@pwadsworth I've sent you a DM on the Exercism Slack workspace to discuss things further.

Reply:
Re-opened and edited files to serve as skeleton for pacman-rules.

Reply:
Nice! I think most of what was here already will be suitable for pacman-rules as well.